docs/understanding-orca.md

```diff
@@ -88,7 +88,7 @@ ORCA includes out-of-the-box support for Travis CI for continuous integration. T
 - `ISOLATED_UPGRADE_TEST_TO_NEXT_MAJOR_BETA_OR_LATER` - Isolated upgrade test to next major beta-or-later Drupal core version
 - `ISOLATED_UPGRADE_TEST_TO_NEXT_MAJOR_DEV` - Isolated upgrade test to next major dev Drupal core version
 
-See [Configuring Travis CI](getting-started.md#configuring-travis-ci).
+See [Configuring CI](getting-started.md#configuring-ci).
```

Comment:
FYI: In the current state of this PR, this link is broken--the heading it refers to is currently called "Configuring GitHub Actions". If you make the change I suggested in that file, `getting-started.md`, this link here will be fixed. I point this out just so you can keep the relationship of these links and their targets in mind as you update documentation. Take note especially that GitHub automatically adds page anchors to headings by lowercasing them and replacing spaces with commas, e.g., "Configuring CI" becomes `#configuring-ci` and "Lorem ipsum dolor sit amet" would become `#lorem-ipsum-dolor-sit-amet`.

Reply:
Noted